### DISCUSSION TITLE
Welcome to wolff Discussions!

### DISCUSSION THREAD

**Discussion by Usbac** [Usbac/Wolff#31] [General]
## 👋🐺 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Be polite, remember that this is a community we build together. 💪

 To get started, comment below with an introduction of yourself and tell us about what you do with this community. 😄

**Comment by siteantipas** [comment]
Proudly powered by Wolff Framework: https://noupia.com

**Reply by Usbac** [reply]
That's a great looking website! A section of 'made with Wolff' will be added to [getwolff.com](https://getwolff.com) once I find the enough amount of public websites made with it. 😃
